tillerinobot/src/main/java/tillerino/tillerinobot/lang/Indonesian.java

```diff
@@ -0,0 +1,271 @@
+	}
+
+	@Override
+	public String help() {
+		return "Hai! Saya adalah bot yang membunuh Tillerino dan mengambil alih akunnya. Hanya bercanda, tetapi aku sering memakai akun ini."
+				+ " [https://twitter.com/Tillerinobot status dan perbaruan]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki perintah]"
+				+ " - [http://ppaddict.tillerino.org/ ppaddict]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki/Contact Kontak]";
+	}
+
+	@Override
+	public String faq() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ Pertanyaan yang sering diajukan (FAQ)]";
+	}
+
+	@Override
+	public String featureRankRestricted(String feature, int minRank, OsuApiUser user) {
+		return "Maaf, sampai saat ini " + feature + " hanya dapat dipakai untuk pengguna yang telah melewati peringkat " + minRank + ".";
+	}
+
+	@Override
+	public String mixedNomodAndMods() {
+		return "Apa yang anda maksud nomod dengan mod?";
+	}
+
+	@Override
+	public String outOfRecommendations() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ#the-bot-says-its-out-of-recommendations-what-do"
+				+ " Saya telah merekomendasikan apa yang saya bisa pikirkan]."
+				+ " Coba gunakan opsi rekomendasi lain atau gunakan !reset. Jika anda tidak yakin, cek !help.";
+	}
+
+	@Override
+	public String notRanked() {
+		return "Sepertinya map itu tidak di rank.";
+	}
+
+	@Override
+	public boolean isChanged() {
+		return false;
+	}
+
+	@Override
+	public void setChanged(boolean changed) {
+
+	}
+
+	@Override
+	public String invalidAccuracy(String acc) {
+		return "Akurasi yang tidak sah: \"" + acc + "\"";
+	}
+
+	@Override
+	public Response optionalCommentOnLanguage(OsuApiUser apiUser) {
+		/*
+		 * TRANSLATION NOTE: This line is sent to the user right after they have
+		 * chosen this Language implementation. The English version refers to
+		 * itself as the default version ("just the way I am"), so translating
+		 * the English message doesn't make any sense.
+		 * 
+		 * Instead, we've been using the line
+		 * "*Translator* helped me learn *Language*." in translations. Replace
+		 * *Translator* with your osu name and *Language* with the name of the
+		 * language that you are translating to, and translate the line into the
+		 * new language. This serves two purposes: It shows that the language
+		 * was changed and gives credit to the translator.
+		 * 
+		 * You don't need to use the line above, and you don't have have to give
+		 * yourself credit, but you should show that the language has changed.
+		 * For example, in the German translation, I just used the line
+		 * "Nichts leichter als das!", which translates literally to
+		 * "Nothing easier than that!", which refers to German being my first
+		 * language.
+		 * 
+		 * Tillerino
+		 * 
+		 * P.S. you can put a link to your profile into the line like this:
+		 * [https://osu.ppy.sh/u/2070907 Tillerino]
+		 */
+		return new Message("Jadi kau suka denganku apa adanya :)");
```

Comment:
this message shouldnt be translated directly, read the comments above it. You don't have to do the suggested thing of crediting yourself but translating it directly doesnt make sense imo